docs/user/HTTP_TRAFFIC_SPLIT.md

```diff
@@ -0,0 +1,111 @@
+# HTTPRoute Traffic Split
+
+The [HTTPRoute][] resource allows one or more `backendRefs` to be provided. Requests will be routed to these upstreams if they match the rules of the HTTPRoute. If a backendRefs configured in an HTTPRoute is invalid, then HTTP responses will be returned with status code `500` for all of the requests that would have been sent to that backend.
+
+## Single backendRef
+
+When a single backendRef is configured in a HTTPRoute, it will receive 100% of the traffic.
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: http-headers
+spec:
+  parentRefs:
+  - name: eg
+  hostnames:
+  - headers.example
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /
+    backendRefs:
+    - group: ""
+      kind: Service
+      name: httpbin
+      port: 80
+EOF
+```
+
+## Multiple backendRefs
+
+If multiple backendRefs are configured, then traffic will be split between the backendRefs equally unless a weight is configured.
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: http-headers
+spec:
+  parentRefs:
+  - name: eg
+  hostnames:
+  - headers.example
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /
+    backendRefs:
+    - group: ""
+      kind: Service
+      name: httpbin
+      port: 80
+    - group: ""
+      kind: Service
+      name: httpbin-2
+      port: 80
+EOF
+```
+
+## Weighted backendRefs
+
+If multiple backendRefs are configured and an un-even traffic split between the backends is desired, then the `weight` field can be used to control the weight of requests to each backend.
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: http-headers
+spec:
+  parentRefs:
+  - name: eg
+  hostnames:
+  - headers.example
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /
+    backendRefs:
+    - group: ""
+      kind: Service
+      name: httpbin
+      port: 80
+      weight: 8
+    - group: ""
+      kind: Service
+      name: httpbin-2
+      port: 80
+      weight: 2
+EOF
+```
+
+In the above example, 80% of the traffic will be sent to the httpbin service, and 20% will be sent to the httpbin-2 service. If the httpbin-2 service does not exist or is invalid, then 80% of the traffic will be sent to the httpbin service, and 20% of the traffic will receive a HTTP response with status code `500`.
```

Comment:
Can you also refer to the upstream docs for additional details on how weight is calculated?